<launch>

  <param name="giskard/robot_description" command="$(find xacro)/xacro '$(find iai_pr2_description)/robots/pr2_with_odom_joints.xacro'" />

  <node pkg="giskardpy" type="base_joint_state_publisher.py" name="base_joint_state_publisher" output="screen"/>

  <node pkg="topic_tools" name="remap_joint_states" type="relay" args="joint_states body/joint_states"/>

  <include file="$(find omni_pose_follower)/launch/omni_pose_follower_map_pr2.launch"/>
  <!-- include file="$(find base_controller)/launch/base_controller_pr2.launch"/ -->

  <node pkg="giskardpy" type="giskard_pr2_real.py" name="giskard" output="screen"/>

  <node pkg="giskardpy" type="interactive_marker.py" name="giskard_interactive_marker" output="screen">
    <rosparam param="enable_self_collision">False</rosparam>
    <rosparam param="interactive_marker_chains">
      - [odom_combined, r_gripper_tool_frame]
      - [base_footprint, l_gripper_tool_frame]
      - [odom_combined, base_footprint]
      - [odom_combined, head_mount_kinect2_rgb_optical_frame]
    </rosparam>
  </node>

</launch>
